packages/ui/src/components/molecules/CardHome.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC } from 'react'
+import styled from 'styled-components'
+import { FlexBox, colors, dimensions, font } from '../../styles'
+import { Text } from '../atoms'
+
+const CardContainer = styled(FlexBox)`
+  flex-direction: column;

```

Comment:
flex-direction se puede poner directamente como propiedad de FlexBox en el componente

Reply:
The same for justify-content below

Reply:
Done!